apps/src/aichat/views/AichatView.tsx

```diff
@@ -14,14 +14,21 @@ import Button from '@cdo/apps/componentLibrary/button/Button';
 import ProjectTemplateWorkspaceIcon from '@cdo/apps/templates/ProjectTemplateWorkspaceIcon';
 import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS, PLATFORMS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import {
+  DialogContext,
+  DialogType,
+} from '@cdo/apps/lab2/views/dialogs/DialogManager';
 const commonI18n = require('@cdo/locale');
 const aichatI18n = require('@cdo/aichat/locale');
 
 import {
+  addChatMessage,
+  resetToDefaultAiCustomizations,
   setStartingAiCustomizations,
   setViewMode,
   clearChatMessages,
   selectAllFieldsHidden,
+  RESET_MODEL_NOTIFICATION,
```

Comment:
Define this constant in this file (or in some constants file)?

Reply:
Chatted offline and we both agree to keep things simpler move constant back to `aichatRedux.ts`.
Discussed that a future refactor could include creating a `utils` file that includes `getNewMessageId`, `latestMessageId`, `getCurrentTimestamp`, etc.